### DISCUSSION TITLE
Add note field

### DISCUSSION THREAD

**Discussion by ghost** [snappysnappydog/Simple-IP-Config#118] [Ideas]
**Version:**
2.9.3

**Expected Behavior:**
free text field with each profile

**Actual Behavior:**
free text field which can hold a description of the network -in my idea- a note of how/where to connect (eg. ssid, which switch/port, vlan)

**Comment by KurtisLiggett** [comment]
Seems like a useful idea.

**Comment by ghost** [comment]
Yes, that might come in handy. Basically all datacenter environments have no DHCP and maintaining a list of how and where to connect is becoming troublesome

**Reply by KurtisLiggett** [reply]
@tonoitp 
Would it make more sense to make this always visible so you can immediately see the notes when switching between profiles? Or should this be more like a popup to reference only when needed?

**Comment by ghost** [comment]
I think it would be nice if it was in the "Profile IP Properties" block. (That however has of-course limited space)
But alternatively a pop-up when clicking the bottom bar will work as well.

(give users a finger, they take the hand -if not the arm- ........... first line always in "Profile IP Properties" block, full text in the pop-up when needed ??? )
